tests/test_network.py

```diff
@@ -45,6 +49,32 @@ def test_load_network_from_string():
     n = pp.network.load_from_string('simple-eu.uct', file_content)
     assert 1 == len(n.get_substations())
 
+def test_load_compressed():
+  with open(DATA_DIR.joinpath('simple-eu.uct'), "rb") as fh:
+      network_data = io.BytesIO(fh.read()).read()
+
+      bz2data = bz2.compress(network_data)
+      n = pp.network.load_from_binary_buffer('simple-eu.uct', io.BytesIO(bz2data))
```

Comment:
@obrix do we really need to specify a name?

Reply:
Not sure, It depends on format read. Still have to figure out the proper parameter to expose because :

* When importing CGMES from one or multiple zip, filename is not important and can be anything : import succeed. So not needed.
* But for data format such as single UCTE file a check is done by the importer to find a element with ".uct" extension in the ReadOnlyMemDataSource data hashmap. So it is not trivial to have a completely generic approach.

Maybe the solution is just to dedicate the multiple bytesIO api to CGMES zipped file import, with no mandatory filename. And then have a single bytesIO import API that has a mandatory file name parameter.

Reply:
Or for the single byteIO api we could add a mandatory format parameter as an enum (UCTE, MAT, etc...). This would avoid the mandatory filename when only the extension is needed

So we could have : 

load_from_binary_buffer(byteBuffer, fomat.UCTE)

load_cgmes_from_binary_buffer([byteBufferZip1, byteBufferZip2...])